onnxruntime/python/onnxruntime_validation.py

```diff
@@ -56,3 +57,90 @@ def check_distro_info():
     else:
         warnings.warn('Unsupported platform (%s). ONNX Runtime supports Linux, macOS and Windows platforms, only.' %
                       __my_system__)
+
+
+def validate_build_package_info():
+    try:
+        from onnxruntime.training.ortmodule import ORTModule # noqa
+        has_ortmodule = True
+    except ImportError:
+        has_ortmodule = False
+    except: # noqa
+        # this may happen if Cuda is not installed
+        has_ortmodule = True
```

Comment:
>has_ortmodule [](http://example.com/codeflow?start=8&length=13)

Seems not quite right...

Reply:
ImportError means there is no ORTModule in onnxruntime.training.ortmodule package. This means has_ortmodule = False. However, if there is ORTModule but there is a different problem such as Cuda not installed, import ortmodule will throw different type of exceptions. In this case, we need to find out local Cuda installation and post warnings.

Reply:
Further refined the code.